server/intelligence-service/scripts/generate_db_models.py

```
@@ -0,0 +1,105 @@
+    password = os.getenv("DB_PASSWORD", "root")
+
+    return f"postgresql+psycopg://{username}:{password}@{host}:{port}/{database}"
+
+
+def generate_db_models(output_file: Path) -> None:
+    """Generate SQLAlchemy models using sqlacodegen."""
+
+    database_url = get_database_url()
+
+    # Get the intelligence service directory (parent of scripts directory)
+    intelligence_service_dir = Path(__file__).parent.parent
```

Ruff (0.11.9)

Local variable `intelligence_service_dir` is assigned to but never used

Remove assignment to unused variable `intelligence_service_dir`

(F841)